Bento/Adapters/TableViewAdapter.swift

```diff
@@ -125,7 +125,8 @@ open class TableViewAdapterBase<SectionID: Hashable, ItemID: Hashable>
 
     @objc(tableView:estimatedHeightForRowAtIndexPath:)
     open func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
-        return store.size(forItemAt: indexPath).map { $0.height + tableView.separatorHeight }
+        return store.size(forItemAt: indexPath, allowEstimation: true)
+            .map { $0.height + tableView.separatorHeight }
```

Comment:
So is it now lazily computes the height?

Reply:
It doesn't compute the height when the `UITableView` asks for the estimated height.

In presence of estimated height, `UITableView` is supposed to call `heightForRowAtIndexPath:` only when the cell is about to enter the prefetching & visible sliding window.